### PR TITLE
check.py fixes

### PR DESCRIPTION
**Pull request by pvelesko**
modules off by default
always build hipInfo

### CODE REVIEW

**Comment by pvelesko** [comment]
Clang crash:

```
PLEASE submit a bug report to https://github.com/llvm/llvm-project/issues/ and include the crash backtrace, preprocessed source, and associated run script.
Stack dump:
0.	Program arguments: /opt/install/llvm/17.0/bin/clang-17 -cc1 -triple x86_64-unknown-linux-gnu -aux-triple spirv64 -emit-obj -disable-free -clear-ast-before-backend -disable-llvm-verifier -discard-value-names -main-file-name standalone_main.cc -mrelocation-model pic -pic-level 2 -fhalf-no-semantic-interposition -mframe-pointer=none -fmath-errno -fno-rounding-math -mconstructor-aliases -funwind-tables=2 -target-cpu x86-64 -target-feature +f16c -tune-cpu generic -debug-info-kind=constructor -dwarf-version=4 -debugger-tuning=gdb -fcoverage-compilation-dir=/opt/actions-runner/_work/chipStar/chipStar/build/catch/catch_tests/hipTestMain -resource-dir /opt/install/llvm/17.0/lib/clang/17 -dependency-file CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o.d -MT catch/catch_tests/hipTestMain/CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o -sys-header-deps -canonical-system-headers -internal-isystem /opt/install/llvm/17.0/lib/clang/17 -idirafter /opt/actions-runner/_work/chipStar/chipStar/build/include -include /opt/actions-runner/_work/chipStar/chipStar/include/hip/spirv_fixups.h -I //opt/actions-runner/_work/chipStar/chipStar/build/include -D KERNELS_PATH=/opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/kernels/ -I /opt/actions-runner/_work/chipStar/chipStar/CHIP -I /opt/actions-runner/_work/chipStar/chipStar/PUBLIC -I /usr/local/lib/../include/level_zero -I /opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/external/Catch2 -I /opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/./include -I /opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/./kernels -I /opt/actions-runner/_work/chipStar/chipStar/build/include -I /opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/external/picojson -D __HIP_PLATFORM_SPIRV__= -I /opt/actions-runner/_work/chipStar/chipStar/include -I /opt/actions-runner/_work/chipStar/chipStar/HIP/include -I /opt/actions-runner/_work/chipStar/chipStar/build/include -I/home/pvelesko/install/intel/opencl/2023.10.02/include -I/opt/install/llvm/17.0/include -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12 -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../include/x86_64-linux-gnu/c++/12 -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/backward -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12 -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../include/x86_64-linux-gnu/c++/12 -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../include/c++/12/backward -internal-isystem /opt/install/llvm/17.0/lib/clang/17/include -internal-isystem /usr/local/include -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../x86_64-linux-gnu/include -internal-externc-isystem /usr/include/x86_64-linux-gnu -internal-externc-isystem /include -internal-externc-isystem /usr/include -internal-isystem /opt/install/llvm/17.0/lib/clang/17/include -internal-isystem /usr/local/include -internal-isystem /usr/lib/gcc/x86_64-linux-gnu/12/../../../../x86_64-linux-gnu/include -internal-externc-isystem /usr/include/x86_64-linux-gnu -internal-externc-isystem /include -internal-externc-isystem /usr/include -O1 -Wno-duplicate-decl-specifier -Wno-tautological-constant-compare -Wno-c++20-extensions -Wno-unused-result -Wno-delete-abstract-non-virtual-dtor -Wno-deprecated-declarations -Wunused-command-line-argument -Wno-format-extra-args -Wall -std=c++17 -fdeprecated-macro -fdebug-compilation-dir=/opt/actions-runner/_work/chipStar/chipStar/build/catch/catch_tests/hipTestMain -ferror-limit 19 -fhip-new-launch-api -fgnuc-version=4.2.1 -fcxx-exceptions -fexceptions -fcuda-include-gpubinary /tmp/standalone_main-86b3e1.hipfb -cuid=d7b952cca3ca0fd7 -fcuda-allow-variadic-functions -faddrsig -D__GCC_HAVE_DWARF2_CFI_ASM=1 -o CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o -x hip /opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/hipTestMain/standalone_main.cc
1.	<eof> parser at end of file
2.	Optimizer
#0 0x00005609bdd9dcb2 llvm::sys::PrintStackTrace(llvm::raw_ostream&, int) (/opt/install/llvm/17.0/bin/clang-17+0x1dcecb2)
#1 0x00005609bdd9b3cb SignalHandler(int) Signals.cpp:0:0
#2 0x00007ff74b04b520 (/lib/x86_64-linux-gnu/libc.so.6+0x42520)
#3 0x00005609cfd8d0e8 
clang++: error: unable to execute command: Segmentation fault (core dumped)
clang++: error: clang frontend command failed due to signal (use -v to see invocation)
clang version 17.0.0 (git@github.com:CHIP-SPV/llvm-project.git 61c6190747f4df6e60eb93b372c4310d779e2886)
Target: x86_64-unknown-linux-gnu
Thread model: posix
InstalledDir: /opt/install/llvm/17.0/bin
clang++: note: diagnostic msg: Error generating preprocessed source(s).

make[3]: *** [catch/catch_tests/hipTestMain/CMakeFiles/Main_Object_Standalone.dir/build.make:76: catch/catch_tests/hipTestMain/CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o] Error 1
failed to execute:/opt/install/llvm/17.0/bin/clang++ -include /opt/actions-runner/_work/chipStar/chipStar/include/hip/spirv_fixups.h -I//opt/actions-runner/_work/chipStar/chipStar/build/include -DKERNELS_PATH="/opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/kernels/" -I/opt/actions-runner/_work/chipStar/chipStar/CHIP -I/opt/actions-runner/_work/chipStar/chipStar/PUBLIC -I/usr/local/lib/../include/level_zero -I/opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/external/Catch2 -I/opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/./include -I/opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/./kernels -I/opt/actions-runner/_work/chipStar/chipStar/build/include -I/opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/external/picojson -Wno-duplicate-decl-specifier -Wno-tautological-constant-compare -Wno-c++20-extensions -Wno-unused-result -Wno-delete-abstract-non-virtual-dtor -Wno-deprecated-declarations -Wunused-command-line-argument --std=c++17 -g -fPIC -Wno-format-extra-args -mf16c -Wall -gdwarf-4 -O1 -std=c++17 -MD -c -x hip /opt/actions-runner/_work/chipStar/chipStar/HIP/tests/catch/hipTestMain/standalone_main.cc -D__HIP_PLATFORM_SPIRV__= --offload=spirv64 -nohipwrapperinc --hip-path=/opt/actions-runner/_work/chipStar/chipStar/build --target=x86_64-unknown-linux-gnu   -I/opt/actions-runner/_work/chipStar/chipStar/include -I/opt/actions-runner/_work/chipStar/chipStar/HIP/include -I/opt/actions-runner/_work/chipStar/chipStar/build/include -o CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o -MT catch/catch_tests/hipTestMain/CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o -MF CMakeFiles/Main_Object_Standalone.dir/standalone_main.cc.o.d
make[2]: *** [CMakeFiles/Makefile2:20330: catch/catch_tests/hipTestMain/CMakeFiles/Main_Object_Standalone.dir/all] Error 2
make[1]: *** [CMakeFiles/Makefile2:5679: catch/CMakeFiles/build_tests.dir/rule] Error 2
make: *** [Makefile:1408: build_tests] Error 2

```

**Comment by pjaaskel** [comment]
Sounds like a fix that is 1.1-material too.

File: scripts/check.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 parser.add_argument('--num-threads', type=int, nargs='?', default=os.cpu_count(), help='Number of threads to use (default: number of cores on the system)')
 parser.add_argument('--num-tries', type=int, nargs='?', default=1, help='Number of tries (default: 1)')
 parser.add_argument('--timeout', type=int, nargs='?', default=200, help='Timeout in seconds (default: 200)')
-parser.add_argument('-m', '--modules', type=str, choices=['on', 'off'], default="on", help='load modulefiles automatically (default: on)')
+parser.add_argument('-m', '--modules', type=str, choices=['on', 'off'], default="off", help='load modulefiles automatically (default: on)')
```
Comment (franz) [inline scripts/check.py:19]: the comment still says "default: on"
